nss-evening-cohort-14/SweetTooth
# Process delete order

## Description
- Order can be processed
- Once Order is processed, a new order is created, 
- Order is deleted then a new order is created,
- GetUnprocessedOrderByUserId creates a new order if there isn't one
- When orderItems are added or updated on shopping page, the order total is updated. 
- counter on the shopping page is based on the quantity of an item 
- when an orderItem quantity is back to zero, the item is deleted. 
- Added basic styling for cart 


#77 
#102 
## Related Issue






## Motivation and Context
Part of essential features for the order


## How Can This Be Tested?
- make sure you have an order with some orderItems
- Click on process Order
- check in database if order has been processed and if new one was created
- create new order through swagger with some orderItems 
- empty cart. check that the order has been deleted, and new one was created.



## Types of changes
